### DISCUSSION TITLE
Documentation.

### DISCUSSION THREAD

**Discussion by db00t** [autobrr/autobrr#813] [General]
I se this is on your webpage and here on this git.

> Software like Radarr and Sonarr utilizes RSS to look for new torrents. RSS feeds are updated regularly, but too slow to let you be a part of what we call the initial swarm of a torrent. This is were autobrr comes into play.

How much slower is RSS???
How fast does it have to be to be in the initial swarm? Seconds? Minutes?
How is autobrr receiving its announces?

**Comment by KyleSanderson** [comment]
RSS is exactly 0s to your interval slower (so, if your interval is 30 minutes, you have a max penalty of 29m59s).
